handbook/demand/README.md

```diff
@@ -51,12 +51,19 @@ The Head of Demand is the DRI for deploying Fleet's event budget, and events are
   - who from Fleet will attend
   - the training plan for Fleet attendees
   - which talk proposal Fleet will submit
-  - all event materials, including printouts, banners, swag given out, and even the clothing worn by fleeties
-  - estimated budget, including sponsorship or airfare, and lodging for attendees
+  - all event materials, including printouts, banners, swag given out, and even the clothing worn by Fleeties
+  - estimated budget, including sponsorship or airfare and lodging for attendees
 2. Set up and attend a 30m meeting with the Head of Business Operations and CEO:
-  - First during this meeting, the Head of Business Operations proposes an event issue for each of the **_current quarter's_** events to get input and any new information or changes from the Head of Demand and CEO. (Events for the current quarter were already decided in a previous event strategy session, so Fleet does not make changes except in extreme circumstances.)
+  - First, during this meeting, the Head of Business Operations proposes an event issue for each of the **_current quarter's_** events to get input and any new information or changes from the Head of Demand and CEO. (Events for the current quarter were already decided in a previous event strategy session, so Fleet does not make changes except in extreme circumstances.)
   - Next, decide which events in the **_following quarter_** the company will invest time or money into.  This includes any event that Fleet pays to send someone to or to sponsor, and even events where Fleet's only involvement is that a fleetie will be giving a talk or otherwise representing the brand.
-  - Finally, qualify or disqualify any newly-entered event ideas by either verifying and setting the buying situation, or removing the event idea from the spreadsheet.
+  - Finally, qualify or disqualify any newly-entered event ideas by either verifying and setting the buying situation or removing the event idea from the spreadsheet.
+
+### Event lead follow up
```

Comment:
@DrewBakerfdm Let's get together re this and below. Good stuff but I feel like it might belong somewhere else (e.g. [communications](https://fleetdm.com/handbook/company/communications#attending-conferences) maybe?)

Reply:
I think it could go in communications, can I just close this PR and add this commit to the communications section under: [https://fleetdm.com/handbook/company/communications#attending-conferences](https://fleetdm.com/handbook/company/communications#attending-conferences) @Sampfluger88

Reply:
Feel free to drop some time on my calendar if you need to!